HackIllinois/Controllers/ViewControllers/HIAdminEventViewController.swift

```diff
@@ -50,10 +49,15 @@ extension HIAdminEventViewController {
 
         let message = "Create a new tracked event \"\(title)\" for \(duration) minutes?"
         let confirmAlertController = UIAlertController(title: "Confirm Tracked Event", message: message, preferredStyle: .alert)
-        confirmAlertController.addAction(
-            UIAlertAction(title: "Yes", style: .default) { _ in
-                self.stylizeFor(.currentlyCreatingEvent)
-                HITrackingService.create(name: title, duration: duration)
+        confirmAlertController.addAction(getConfirmAlertAction(title: title, duration: duration))
+        confirmAlertController.addAction(UIAlertAction(title: "No", style: .cancel, handler: nil))
+        self.present(confirmAlertController, animated: true, completion: nil)
+    }
+
+    private func getConfirmAlertAction(title: String, duration: Int) -> UIAlertAction {
+        return UIAlertAction(title: "Yes", style: .default) { _ in
+            self.stylizeFor(.currentlyCreatingEvent)
+            HITrackingService.create(name: title, duration: duration)
```

Comment:
a better (more cocoa like) name for this would be confirmAlertActionWith(title: String, duration: Int)